### PR TITLE
Dependency updates, tslinting, moved to @types an vscode step-through debugging

### PR DESCRIPTION
**Pull request by moanrose**

### CODE REVIEW

File: typings.json [hunk 1]
```diff
@@ -1,18 +0,0 @@
-{
```
Comment (brechtbilliet) [inline typings.json]: the tsconfig.json should be updated accordingly :)

Comment (moanrose) [inline typings.json]: Enjoy

**Comment by NathanWalker** [comment]
Any reason this PR is held up? Also curious if webpack 2.0 update may be in the future here?

**Comment by brechtbilliet** [comment]
Sorry guys, I was real busy. Merged :)

**Pull request merged**